dev-7/dev-7/TypesOfXML.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace dev_7
+{
+    /// <summary>
+    /// Enum of XML-file types.
+    /// </summary>
+    enum TypesOfXML
+    {
+        Car = 0,
```

Comment:
you are using 0 for command line arguments. but it is not declared. You should always write comments in this cases